### ISSUE TITLE
source_morphology fails with TypeError

### ISSUE THREAD

**Issue by danjampro** [vrodgom/statmorph#9]
```
>>> source_morphs = statmorph.source_morphology(data, segmap, weightmap=np.ones_like(data))

TypeError                                 Traceback (most recent call last)
<ipython-input-10-6e5293b8e978> in <module>
----> 1 source_morphs = statmorph.source_morphology(data, segmap, weightmap=np.ones_like(data))

~/anaconda3/lib/python3.7/site-packages/statmorph/statmorph.py in source_morphology(image, segmap, **kwargs)
   2472     sources_morph = []
   2473     for label in segmap.labels:
-> 2474         sources_morph.append(SourceMorphology(image, segmap, label, **kwargs))
   2475         print('Finished processing source %d.\n' % (label))
   2476

~/anaconda3/lib/python3.7/site-packages/statmorph/statmorph.py in __init__(self, image, segmap, label, mask, weightmap, gain, psf, cutout_extent, min_cutout_size, n_sigma_outlier, annulus_width, eta, petro_fraction_gini, skybox_size, petro_extent_cas, petro_fraction_cas, boxcar_size_mid, niter_bh_mid, sigma_mid, petro_extent_flux, boxcar_size_shape_asym, sersic_maxiter, segmap_overlap_ratio)
    443
    444         # For simplicity, evaluate all "lazy" properties at once:
--> 445         self._calculate_morphology()
    446
    447         # Check if image is background-subtracted; set flag=1 if not.

~/anaconda3/lib/python3.7/site-packages/statmorph/statmorph.py in _calculate_morphology(self)
    544         ]
    545         for q in quantities:
--> 546             tmp = self[q]
    547
    548     def _check_segmaps(self):

~/anaconda3/lib/python3.7/site-packages/statmorph/statmorph.py in __getitem__(self, key)
    457
    458     def __getitem__(self, key):
--> 459         return getattr(self, key)
    460
    461     def _get_badpixels(self, image):

~/anaconda3/lib/python3.7/site-packages/astropy/utils/decorators.py in __get__(self, obj, owner)
    756                 return val
    757             else:
--> 758                 val = self.fget(obj)
    759                 obj.__dict__[self._key] = val
    760                 return val

~/anaconda3/lib/python3.7/site-packages/statmorph/statmorph.py in xc_asymmetry(self)
   1563         relative to the original image.
   1564         """
-> 1565         return self.xmin_stamp + self._asymmetry_center[0]
   1566
   1567     @lazyproperty

~/anaconda3/lib/python3.7/site-packages/astropy/utils/decorators.py in __get__(self, obj, owner)
    756                 return val
    757             else:
--> 758                 val = self.fget(obj)
    759                 obj.__dict__[self._key] = val
    760                 return val

~/anaconda3/lib/python3.7/site-packages/statmorph/statmorph.py in _asymmetry_center(self)
   1540         center_asym = opt.fmin(self._asymmetry_function, center_0,
   1541                                args=(self._cutout_stamp_maskzeroed, 'cas'),
-> 1542                                xtol=1e-6, disp=0)
   1543
   1544         # Check if flag was activated by _asymmetry_function:

~/anaconda3/lib/python3.7/site-packages/scipy/optimize/optimize.py in fmin(func, x0, args, xtol, ftol, maxiter, maxfun, full_output, disp, retall, callback, initial_simplex)
    441             'initial_simplex': initial_simplex}
    442
--> 443     res = _minimize_neldermead(func, x0, args, callback=callback, **opts)
    444     if full_output:
    445         retlist = res['x'], res['fun'], res['nit'], res['nfev'], res['status']

~/anaconda3/lib/python3.7/site-packages/scipy/optimize/optimize.py in _minimize_neldermead(func, x0, args, callback, maxiter, maxfev, disp, return_all, initial_simplex, xatol, fatol, adaptive, **unknown_options)
    584
    585     for k in range(N + 1):
--> 586         fsim[k] = func(sim[k])
    587
    588     ind = numpy.argsort(fsim)

~/anaconda3/lib/python3.7/site-packages/scipy/optimize/optimize.py in function_wrapper(*wrapper_args)
    325     def function_wrapper(*wrapper_args):
    326         ncalls[0] += 1
--> 327         return function(*(wrapper_args + args))
    328
    329     return ncalls, function_wrapper

~/anaconda3/lib/python3.7/site-packages/statmorph/statmorph.py in _asymmetry_function(self, center, image, kind)
   1486         # Create aperture for the chosen kind of asymmetry
   1487         if kind == 'cas':
-> 1488             r = self._petro_extent_cas * self._rpetro_circ_centroid
   1489             ap = photutils.CircularAperture(center, r)
   1490         elif kind == 'outer':

~/anaconda3/lib/python3.7/site-packages/astropy/utils/decorators.py in __get__(self, obj, owner)
    756                 return val
    757             else:
--> 758                 val = self.fget(obj)
    759                 obj.__dict__[self._key] = val
    760                 return val

~/anaconda3/lib/python3.7/site-packages/statmorph/statmorph.py in _rpetro_circ_centroid(self)
   1008         """
   1009         center = np.array([self._xc_stamp, self._yc_stamp])
-> 1010         return self._rpetro_circ_generic(center)
   1011
   1012     @lazyproperty

~/anaconda3/lib/python3.7/site-packages/statmorph/statmorph.py in _rpetro_circ_generic(self, center)
    978                               AstropyUserWarning)
    979                 self.flag = 1
--> 980             curval = self._petrosian_function_circ(r, center)
    981             if curval == 0:
    982                 warnings.warn('[rpetro_circ] We found rpetro by pure chance!',

~/anaconda3/lib/python3.7/site-packages/statmorph/statmorph.py in _petrosian_function_circ(self, r, center)
    937         # Force mean fluxes to be positive:
    938         circ_annulus_mean_flux = np.abs(_aperture_mean_nomask(
--> 939             circ_annulus, image, method='exact'))
    940         circ_aperture_mean_flux = np.abs(_aperture_mean_nomask(
    941             circ_aperture, image, method='exact'))

~/anaconda3/lib/python3.7/site-packages/statmorph/statmorph.py in _aperture_mean_nomask(ap, image, **kwargs)
     90         return ap.do_photometry(image, **kwargs)[0][0] / ap.area()
     91     else:
---> 92         return ap.do_photometry(image, **kwargs)[0][0] / ap.area
     93
     94 def _fraction_of_total_function_circ(r, image, center, fraction, total_sum):

TypeError: unsupported operand type(s) for /: 'float' and 'method'
```

**Comment by danjampro** [comment]
Solved by updating `photutils` from `0.7.dev3394` to `0.7.2`.

**Comment by vrodgom** [comment]
OK, glad to hear that updating `photutils` solved the issue.
